### PR TITLE
Updating Notebooks-Spelling Errors

### PR DESCRIPTION
**Pull request by ashwani-rathee**
Updated Tutorial from 0-11 
Excluded (Tutorial-9,A1,A2,A3,A4)<--No changes made
The Excluded ones need to be updated separately..
I excluded Tutorial-9 as I wasn't able to make sense of GOOGLE-API-KEY error..
I excluded A1,A2, A3,A4 as I was told by @bryevdv that
```
Some of the appendices use outside libraries, and may not be up to date. I would suggest skipping them for now and making a separate issue to update them.
```
In an earlier version of making changes,I did make changes to those appendices too, and converted them..They all worked fine and were converted fine. Will make a separate issue for them if necessary..

In my opinion, the tutorials need a thorough review and need to include more engaging and interesting examples. Keeping them updated is just a temporary fix.

### CODE REVIEW

File: tutorial/02 - Styling and Theming.ipynb [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "Collapsed": "false"
    },
    "source": [
-    "In this chapter we will learn how to configure various visual aspects of our plots, and how to find out more about what an be configured. "
+    "In this chapter,we will learn how to configure various visual aspects of our plots, and how to find out more about what can be configured. "
```
Comment (bryevdv) [inline tutorial/02 - Styling and Theming.ipynb:34]: missing a space after the comma, though my personal preference is just not to have a comma there at all.

File: tutorial/02 - Styling and Theming.ipynb [hunk 2]
```diff
@@ -885,7 +881,7 @@
    "source": [
     "# Axes\n",
     "\n",
-    "Next we will take a look at [stlying of Axes](https://bokeh.pydata.org/en/latest/docs/user_guide/styling.html#axes).\n",
+    "Next,we will take a look at the [styling of Axes](https://bokeh.pydata.org/en/latest/docs/user_guide/styling.html#axes).\n",
```
Comment (bryevdv) [inline tutorial/02 - Styling and Theming.ipynb:884]: missing space after comma, but again I don't think a comma here is necessary. I also don't think the added "the" is necessary..

File: tutorial/02 - Styling and Theming.ipynb [hunk 3]
```diff
@@ -1121,9 +1117,9 @@
...
+    "All Bokeh axes have a `formatter` property, whose value is a `TickFormatter` object that Bokeh uses to format the ticks displayed by that axis. Bokeh will configure default tick formatters for numeric, datetime, or categorical axes. But often we would like to customize the appearance of tick labels. This can be accomplished by changing properties on the default formatter that Bokeh chooses, or by replacing the formatter with a new type entirely. \n",
     "\n",
-    "Let's first look at changing the properties of a default formatter. The default datetime formatter is configured to show *month/day* when the axis is on the scale of days. If would like the also always show the year, we can change the `days` property to a format that includes the year, as done below. "
+    "Let's first look at changing the properties of a default formatter. The default datetime formatter is configured to show *month/day* when the axis is on the scale of days. If you would like to also show the year, we can change the `days` property to a format that includes the year, as done below. "
```
Comment (bryevdv) [inline tutorial/02 - Styling and Theming.ipynb:1122]: > If you would like to..., we can change

is a bit awkward with mixed subjects. Suggest:

> If we would like to ...

File: tutorial/04 - Adding Annotations.ipynb [hunk 4]
```diff
@@ -694,7 +690,7 @@
    "source": [
     "## LabelSet\n",
     "\n",
-    "The `LabelSet` annotation allows you to create many labels at once, for instance if you want to label an entire set of scatter markers. They are similar to `Label`, but they can also \n",
+    "The `LabelSet` annotation allows you to create many labels at once, for instance,if you want to label an entire set of scatter markers. They are similar to `Label`, but they can also \n",
```
Comment (bryevdv) [inline tutorial/04 - Adding Annotations.ipynb:693]: Missing space after comma. Also suggest terminating the previous clause:
> The `LabelSet` annotation allows you to create many labels at once. For instance, if ...

File: tutorial/05 - Presentation Layouts.ipynb [hunk 5]
```diff
@@ -362,7 +358,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the previous chapters we started to learn how to create single plots using differnet kinds of data. But we often want to plot more than one thing. Bokeh plots can be individually embedded in HTML documents, but it's often easier to\n",
+    "In the previous chapters,we started to learn how to create single plots using different kinds of data. But we often want to plot more than one thing. Bokeh plots can be individually embedded in HTML documents, but it's often easier to\n",
```
Comment (bryevdv) [inline tutorial/05 - Presentation Layouts.ipynb:361]: missing comma again. Seems to be a consistent problem, I won't not any future occurrences explicitly but all changes should be checked that commas have spaces after them.

File: tutorial/05 - Presentation Layouts.ipynb [hunk 6]
```diff
@@ -383,7 +379,7 @@
    "metadata": {},
    "source": [
     "# Rows and Columns\n",
-    "The `bokeh.layouts` modules provides the ``row`` and ``column`` functions to arrange plot objects in vertical or horizontal layouts. Below is an example of three plots arranged in a row."
+    "The `bokeh.layouts` modules provide the ``row`` and ``column`` functions to arrange plot objects in vertical or horizontal layouts. Below is an example of three plots arranged in a row."
```
Comment (bryevdv) [inline tutorial/05 - Presentation Layouts.ipynb:382]: `bokeh.layouts` is a single module, so the proper fix here is to un-pluralize "modules":
> The `bokeh.layouts` module provides the

File: tutorial/11 - Running Bokeh Applications.ipynb [hunk 7]
```diff
@@ -27,20 +27,339 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The architecture of Bokeh is such that high-level “model objects” (representing things like plots, ranges, axes, glyphs, etc.) are created in Python, and then converted to a JSON format that is consumed by the client library, BokehJS. Using the Bokeh Server, it is possible to keep the “model objects” in python and in the browser in sync with one another, creating powerful capabilities:\n",
+    "The architecture of Bokeh is such that high-level “model objects” (representing things like plots, ranges, axes, glyphs, etc.) are created in Python, and then converted to a JSON format that is consumed by the client library, BokehJS. Using the Bokeh Server, it is possible to keep the “model objects” in python and the browser in sync with one another, creating powerful capabilities:\n",
```
Comment (bryevdv) [inline tutorial/11 - Running Bokeh Applications.ipynb:30]: I would prefer to keep the parallel sentence structure and keep "in" in both sides of the conjunction to emphasize the location where models are (and that they are distinct)

**Comment by bryevdv** [comment]
@ashwani-rathee I've left some comments above FYI we will be releasing 2.1 soon so we should probably wait to merge this until that is done, and then re-eval one more time to get the latest version used..

**Comment by ashwani-rathee** [comment]
@bryevdv ok,that sounds good.will improve on it..

**Comment by bryevdv** [comment]
@ashwani-rathee did you still plan to work on this PR?

**Comment by ashwani-rathee** [comment]
@bryevdv yes

**Comment by bryevdv** [comment]
@ashwani-rathee this got closed because we moved the default branch to `main`, please feel to re-submit with the discussed changes against `main`

**Pull request closed without merging**